src/__tests__/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -789,6 +789,7 @@ exports[`snapshots renders a menu that contains an item to add to the menu 1`] =
       >
         <li
           aria-labelledby="0--label "
+          aria-selected={false}
```

Comment:
It looks like Autocomplete might have had a similar issue with list items not having `aria-selected` attributes, and this fix applies here as well.